www/content/installation/index.md

```diff
@@ -108,13 +108,28 @@ valeur du raccourci Global auto-type.
 
 ### Linux
 
+#### Gnome
 Avec certains bureaux (Gnome notamment), la touche Typo [★]{.odk} ne fonctionne
 que si Ergo‑L est défini comme disposition par défaut, c’est-à-dire en haut de
 la liste dans les préférences clavier.
 
 [Wezterm][] ajoute un `o` à chaque pression de la touche Typo [★]{.odk}. C’est une
 [régression connue][wezterm-bug], [un patch est en cours][wezterm-patch].
 
+#### Ubuntu
+Sous Ubuntu 24.04, il est possible que la touche Typo [★]{.odk} soit active sur
+deux caractères consécutifs.
+Ainsi, `★ss` donne ainsi `éé` plutôt que `és`.
+Pour éviter cela, il faut ajouter les variables d’environnement suivantes dans
+le fichier `/etc/environment` :
+```
+INPUT_METHOD=ibus
+GTK_IM_MODULE=ibus
+QT_IM_MODULE=ibus
+XMODIFIERS=@im=ibus
+```
+
+
```

Comment:
Is an issue created for this ?
Some people was aking on Discord [link1](https://discord.com/channels/1046720208171175946/1046720208619974658/1291384763449610361) I’m not sure yet if it’s related to Ubuntu or just a similarity

Reply:
Edit : i need to learn to read, not really the same things